src/io/export_pov.cc

```diff
@@ -0,0 +1,114 @@
+/*
+ *  OpenSCAD (www.openscad.org)
+ *  Copyright (C) 2021      Konstantin Podsvirov <konstantin@podsvirov.pro>
+ *         Povray 2024      Folkert van Heusden  <mail@vanheusden.com>
+ *
+ *  This program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2 of the License, or
+ *  (at your option) any later version.
+ *
+ *  As a special exception, you have permission to link this program
+ *  with the CGAL library and distribute executables, as long as you
+ *  follow the requirements of the GNU GPL in regard to all of the
+ *  software in the executable aside from CGAL.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program; if not, write to the Free Software
+ *  Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+ *
+ */
+
+#include "export.h"
+
+#include "PolySet.h"
+#include "PolySetUtils.h"
+
+
+void export_pov(const std::shared_ptr<const Geometry>& geom, std::ostream& output, const ExportInfo& exportInfo)
+{
+  auto ps = PolySetUtils::getGeometryAsPolySet(geom);
+  if (Feature::ExperimentalPredictibleOutput.is_enabled()) {
+    ps = createSortedPolySet(*ps);
+  }
+
+  output << "// Generated by OpenSCAD!\n";
+  output << "// Source file: " << exportInfo.sourceFileName << "\n\n";
+
+  output << "#version 3.7;\n";
+  output << "global_settings { assumed_gamma 1.0 }\n";
+  output << "#declare MATERIAL=finish { specular 0.5 roughness 0.001 reflection{0 0.63 fresnel} ambient 0 diffuse 0.6 conserve_energy }\n";
+  output << "#declare MATERIAL_INT=interior{ior 1.32}\n";
+
+  auto has_color = !ps->color_indices.empty();
+
+  for (size_t polygon_index=0; polygon_index<ps->indices.size(); polygon_index++) {
+    const auto &polygon = ps->indices[polygon_index];
+    output << "polygon { " << polygon.size() + 1 << ", \n";
+    for (size_t i=0; i<polygon.size(); i++) {
+      if (i)
+        output << ", ";
+      const auto & x = ps->vertices[polygon[i]].x();
+      const auto & y = ps->vertices[polygon[i]].y();
+      const auto & z = ps->vertices[polygon[i]].z();
+      output << "<" << x << ", " << y << ", " << z << ">";
+    }
+    output << ", <" << ps->vertices[polygon[0]].x() << ", " << ps->vertices[polygon[0]].y() << ", " << ps->vertices[polygon[0]].z() << ">";
+    float r = 1.0, g = 1.0, b = 1.0, a = 0.0;
```

Comment:
Is white a good default, or should we use the color from the current OpenSCAD color scheme?